frontend/package.json

```diff
@@ -44,7 +44,21 @@
     "rimraf": "^3.0.2",
     "semver": "^7.3.4",
     "typescript": "^4.1.3",
-    "xterm": "^4.10.0"
+    "xterm": "^4.10.0",
+    "@iconify/icons-mdi": "^1.1.10",
+    "@iconify/react": "^1.1.3",
+    "@kinvolk/eslint-config": "^0.4.1",
+    "@typescript-eslint/eslint-plugin": "^4.14.2",
+    "@typescript-eslint/parser": "^4.14.2",
+    "eslint": "^7.19.0",
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-react": "^7.22.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-plugin-simple-import-sort": "^7.0.0",
+    "@types/js-base64": "^2.3.1",
+    "@types/js-yaml": "^3.12.4",
+    "prettier": "^2.2.1"
```

Comment:
I don't think these belong to deps, right?
This means that we will ship these as dependencies when we do production builds, and they won't be used at all, unless I am missing something?
(About the `iconify`, it feels like it should be a dependency, but the way it works and how it's recommended, is that it can stay as a dev dependency AFAIU)

Reply:
It just means these dependencies are required for doing production builds. For example, if you remove the iconify icons from there it fails to compile (with `npm install --only=prod && npm run build`).